styles/dialog/fight.scss

```diff
@@ -101,37 +181,38 @@
                 color: black;
                 border-radius: 3px;
             }
-
-            .remove-fighter {
-                width: 1.5em;
-                height: 1.5em;
-                padding: 1px;
-                line-height: 1.3em;
-                color: #800;
-            }
-
-            .actor-name {
-                font-size: 1.125em;
-                font-weight: bold;
-                width: calc(100% - 1.5em);
-                &:hover {
+     
+            .fighter-action-panel {
+                display: flex;
+                flex-direction: row;
+                .remove-fighter {
+                    width: 1.5em;
+                    height: 1.5em;
+                    padding: 1px;
+                    line-height: 1.3em;
                     color: #800;
                 }
-            }
-            .portrait {
-                width: 80px; height: 80px;
-                border: 0;
-                object-fit: cover;
-            }
-            .details {
-                width: calc(95% - 80px);
-                margin-left: 2%;
-                select {
-                    width: 100%;
+                .actor-name {
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+                    font-size: 1.125em;
+                    font-weight: bold;
+                    flex-grow: 1;
+                    padding-right: 1.5em;
```

Comment:
Not sure about this, especially when it comes to the player only view.
![image](https://user-images.githubusercontent.com/5915880/104644704-5c012380-566b-11eb-91ba-5810dba676fd.png)

Reply:
It's the solution for the issue you've had before. Text is now always to center and not colliding with remove icon